Makefile

```diff
@@ -22,21 +22,21 @@ define generate_sources
 	@cd plugins/defaultplugins/l2plugin && go generate
 	@cd plugins/defaultplugins/l3plugin && go generate
 	@cd plugins/defaultplugins/l4plugin && go generate
-	@cd plugins/defaultplugins/aclplugin/bin_api/acl && pkgreflect
-	@cd plugins/defaultplugins/ifplugin/bin_api/af_packet && pkgreflect
-	@cd plugins/defaultplugins/ifplugin/bin_api/bfd && pkgreflect
-	@cd plugins/defaultplugins/ifplugin/bin_api/interfaces && pkgreflect
-	@cd plugins/defaultplugins/ifplugin/bin_api/ip && pkgreflect
-	@cd plugins/defaultplugins/ifplugin/bin_api/memif && pkgreflect
-	@cd plugins/defaultplugins/ifplugin/bin_api/stats && pkgreflect
-	@cd plugins/defaultplugins/ifplugin/bin_api/tap && pkgreflect
-	@cd plugins/defaultplugins/ifplugin/bin_api/tapv2 && pkgreflect
-	@cd plugins/defaultplugins/ifplugin/bin_api/vpe && pkgreflect
-	@cd plugins/defaultplugins/ifplugin/bin_api/vxlan && pkgreflect
-	@cd plugins/defaultplugins/l2plugin/bin_api/l2 && pkgreflect
-	@cd plugins/defaultplugins/l2plugin/bin_api/vpe && pkgreflect
-	@cd plugins/defaultplugins/l3plugin/bin_api/ip && pkgreflect
-	@cd plugins/defaultplugins/l4plugin/bin_api/session && pkgreflect
+	@cd plugins/defaultplugins/common/bin_api/acl && pkgreflect
+	@cd plugins/defaultplugins/common/bin_api/af_packet && pkgreflect
+	@cd plugins/defaultplugins/common/bin_api/bfd && pkgreflect
+	@cd plugins/defaultplugins/common/bin_api/interfaces && pkgreflect
+	@cd plugins/defaultplugins/common/bin_api/ip && pkgreflect
+	@cd plugins/defaultplugins/common/bin_api/memif && pkgreflect
+	@cd plugins/defaultplugins/common/bin_api/stats && pkgreflect
+	@cd plugins/defaultplugins/common/bin_api/tap && pkgreflect
+	@cd plugins/defaultplugins/common/bin_api/tapv2 && pkgreflect
+	@cd plugins/defaultplugins/common/bin_api/vpe && pkgreflect
+	@cd plugins/defaultplugins/common/bin_api/vxlan && pkgreflect
+	@cd plugins/defaultplugins/common/bin_api/l2 && pkgreflect
+	@cd plugins/defaultplugins/common/bin_api/vpe && pkgreflect
+	@cd plugins/defaultplugins/common/bin_api/ip && pkgreflect
```

Comment:
This is duplicated. Binary API for `ip` is already mentioned. Best would be to keep these sorted alphabetically.